docs/web_reference.rst

```diff
@@ -1103,12 +1103,13 @@ passing it into *template engine* for example::
 
    url = app.router['route_name'].url(query={'a': 1, 'b': 2})
 
-There are three concrete route classes:* :class:`DynamicRoute` for
-urls with :ref:`variable pathes<aiohttp-web-variable-handler>` spec.
+There are three concrete route classes:
 
+* :class:`DynamicRoute` for urls with :ref:`variable 
```

Comment:
Isn't it the duplicate for https://github.com/KeepSafe/aiohttp/pull/522/files#diff-ac2fa3303564711b0fb8ba55a72d7d2dL1113 ???